TYPO3.Flow/Documentation/Quickstart/index.rst

```diff
@@ -425,7 +425,7 @@ can you imagine what they do? ::
 
     use Acme\Demo\Domain\Model\CoffeeBean;
     use Acme\Demo\Domain\Repository\CoffeeBeanRepository;
-    
```

Comment:
No problem the other empty code lines are also not indented, so that's just an additional fix :)